documentation/components/badge.html

```diff
@@ -0,0 +1,291 @@
+                    <code class="text-primary fs-6">div</code> as
+                    <code class="text-primary fs-6">badge-container</code> and then inside the 
+                    <code class="text-primary fs-6">div</code> place a icon followed by a 
+                    <code class="text-primary fs-6">span</code> with class name 
+                    <code class="text-primary fs-6">badge icon-badge</code>. This gives an icon with a badge on.
+                </p>
+
+                <!-- Badge on icon example -->
+                <div class="component-example-container p-5">
+                    <div class="badge-container">
+                        <i class="fas fa-home"></i>
+                        <span class="badge icon-badge">15</span>
+                    </div>
+                </div>
+
+                <!-- Badge on icon example code-->
+                <div class="code-container">
+                    <pre>
+                        <code class="language-html">
+                            &lt;div class="badge-container">
+                                &lt;i class="fas fa-home">&lt;/i>
+                                &lt;span class="badge icon-badge">15&lt;/span>
+                            &lt;/div>
+                        </code>
+                    </pre>
+                </div>
+
+                <!-- Badge on avatar -->
+                <h3 id="badge-on-avatars" class="my-2">Badge on Avatars</h3>
+                <p class="fw-light mb-4">
+                    To use badge on an avatar use a container as
+                    <code class="text-primary fs-6">badge-container</code> and then inside the 
+                    <code class="text-primary fs-6">div</code> place a avatar followed by a 
+                    <code class="text-primary fs-6">span</code> with class name 
+                    <code class="text-primary fs-6">badge avatar-badge</code>. This gives an avatar with a badge on.
+                </p>
+
+                <!-- Badge on avatar example -->
+                <div class="component-example-container p-5">
+                    <figure class="avatar avatar-xl badge-container">
+                        <img src="../../assets/avatar.jpg" alt=""></img>
+                        <span class="badge avatar-badge"></span>
+                    </figure>
+                    <figure class="avatar avatar-lg badge-container">
+                        <img src="../../assets/avatar.jpg" alt=""></img>
+                        <span class="badge avatar-badge"></span>
+                    </figure>
+                    <figure class="avatar avatar-md badge-container">
+                        <img src="../../assets/avatar.jpg" alt=""></img>
+                        <span class="badge avatar-badge"></span>
+                    </figure>
+                    <figure class="avatar avatar-sm badge-container">
+                        <img src="../../assets/avatar.jpg" alt=""></img>
```

Comment:
Add `alt` attributes, it helps in accessibility

Reply:
Okay sure.